gitlab/resource_gitlab_deploy_token_test.go

```diff
@@ -165,3 +165,44 @@ resource "gitlab_deploy_token" "foo" {
 }
   `, rInt, rInt)
 }
+
+type expiresAtSuppressFuncTest struct {
+	description string
+	old         string
+	new         string
+	expected    bool
+}
+
+func TestExpiresAtSuppressFunc(t *testing.T) {
+	testcases := []expiresAtSuppressFuncTest{
+		{
+			description: "same dates without millis",
+			old:         "2025-03-14T00:00:00Z",
+			new:         "2025-03-14T00:00:00Z",
+			expected:    true,
+		}, {
+			description: "different date without millis",
+			old:         "2025-03-14T00:00:00Z",
+			new:         "2025-03-14T11:11:11Z",
+			expected:    false,
+		}, {
+			description: "same date with and without millis",
+			old:         "2025-03-14T00:00:00Z",
+			new:         "2025-03-14T00:00:00.000Z",
+			expected:    true,
+		}, {
+			description: "cannot parse new date",
+			old:         "2025-03-14T00:00:00Z",
+			new:         "invalid-date",
+			expected:    false,
+		},
+	}
+
+	for _, test := range testcases {
```

Comment:
Small nitpick for next time you're writing a table test -- It's better to use `t.Run` to create a new subtest for each case in the table, so that all cases will be evaluated instead of stopping on the first failure. https://blog.golang.org/subtests#TOC_4.

Reply:
Thanks for the feedback. That's a valid nitpick. I'll update the PR later today.

Reply:
done. :)